package.json

```diff
@@ -44,14 +44,12 @@
     "turbo": "~1.3.2"
   },
   "resolutions": {
-    "braces": "^2.3.1",
     "hosted-git-info": ">=3.0.8",
     "jimp": ">=0.16.1",
     "trim-newlines": "^3.0.1",
     "minimatch": "^3.0.5",
     "glob-parent": "^6.0.1",
     "jpeg-js": "^0.4.3",
-    "set-value": "^4.1.0",
```

Comment:
This is also not needed?

Reply:
yup. it was coming from one of the transitive deps which no longer requires it, so the resolution is not needed
![](https://user-images.githubusercontent.com/9084735/204779939-7139b864-f0ca-48af-87ae-ee815ebb9c46.png)